workflow.md

```diff
@@ -241,3 +257,56 @@ document, this is to be attached as comment.
 All tasks on current sprint are to be assigned an estimation and properly
 transition according to real state
 
+
+
+## Release managament
+
+This section describes our release practices.
+
+### Fields in YT
+
+We have a number of special-purpose fields in YT for release management:
+
+* Affected versions
+   + For bug reports/testcase failures, should contain a list of versions, for which issue was approved to reproduce.
+* Affected builds
+   + For bug reports/testcase failures, should contain a list of builds, for which issue was approved to reproduce.
+* Target versions
+   + Version, within which we are to deliver the patch associated with issue.
+   
+### Release processing
+
+Following steps should be executed to release version `X = Maj.Min.Fix`:
+
+1. Check issues assigned to target version `X` are all in *Wait for build* state
+
+   Issues should be moved to different version if and only if no commits were pushed to branch corresponding to `X`.
+
+   In case some commits were pushed, some work remained, issue should be split to two (one assigned to next version).
+   Commit revert may also be an option.
+
+2. Extract list of commits from git commit history, i.e. all commits:
+   * From tag of version `Maj.Min.(Fix-1)` if `Fix > 0`.
+   * From tag of version `Maj.(Min - 1).0` if `Fix == 0`.
```

Comment:
I think it's wrong. It should be `Maj.(Min - 1).A` instead of `Maj.(Min - 1).0`, where are is maximum `i` such that `Maj.(Min - 1).i` exists.